src/SFML/Audio/SoundRecorder.cpp

```diff
@@ -203,7 +211,7 @@ bool SoundRecorder::setDevice(const std::string& name)
 
         // Start the capture in a new thread, to avoid blocking the main thread
         m_isCapturing = true;
-        m_thread.launch();
+        m_thread.emplace(&SoundRecorder::record, this);
```

Comment:
I don't think it is worth introducing a helper function here.

Reply:
It's mostly about the name, "launch capturing thread" is more expressive than "emplace record function", then we don't need extra comments. It would also be symmetrical to `awaitCapturingThread()` (so the `m_isCapturing = true` should be part of it).

Reply:
I see, I think it is worthwhile doing if we include `m_isCapturing = true`. I will make that change.